Makefile

```diff
@@ -1,12 +1,14 @@
 DESTDIR :=
 BINDIR := /usr/local/bin
```

Comment:
just a note, it's not part of your PR, but this BINDIR definition should be changed to have a PREFIX. We should be able to specify a PREFIX for our components.